src/measure_theory/function/conditional_expectation.lean

```diff
@@ -2166,6 +2166,37 @@ begin
       (signed_measure.measurable_rn_deriv _ _).strongly_measurable },
 end
 
+lemma integrable.snorm_one_condexp_le_snorm [is_finite_measure μ]
```

Comment:
The integrability hypothesis is not needed. If the function is not integrable, the conditional expectation is zero and your inequality is `0 ≤ snorm f 1 μ`, which is true.
Also I think you should not need `is_finite_measure`: sigma-finite with respect to m should be enough. And in fact, if you don't have `hm` or that sigma-integrability, the conditional expectation is also zero and the result is true.

Also please add the following docstring:
```suggestion
/-- TODO: this should be generalized and proved using Jensen's inequality
for the conditional expectation (not in mathlib yet) .-/
lemma integrable.snorm_one_condexp_le_snorm [is_finite_measure μ]
```